linter_exclusions.yml

```diff
@@ -253,6 +253,9 @@ aks create:
     workspace_resource_id:
       rule_exclusions:
       - option_length_too_long
+    node_osdisk_diskencryptionset_id:
```

Comment:
You can provide one more shorter abbreviation like `options_list=['--node-osdisk-diskencryptionset-id', '--disk-encryp-set' ]`. Is there any reason to suppress this linter rule?

Reply:
@MyronFanQiu since some customers are already using `--node-osdisk-diskencryptionset-id`, we want to keep the same name. `--disk-encryptset` is not a good abbreviation, must add `node-osdisk`. Anyway, let's keep the original naming. thanks.

Reply:
@andyzhangx You can keep the original naming. But it's too long and not user friendly. You should add one more shorter abbreviation. User can see two options for this argument.

Reply:
fixed by add a new short name `-d`